server/app/views/applicant/TrustedIntermediaryDashboardView.java

```diff
@@ -55,12 +63,15 @@
 public class TrustedIntermediaryDashboardView extends BaseHtmlView {
   private final ApplicantLayout layout;
   private final DateConverter dateConverter;
+  private final ProgramRepository programRepository;
   public static final String OPTIONAL_INDICATOR = " (optional)";
 
   @Inject
-  public TrustedIntermediaryDashboardView(ApplicantLayout layout, DateConverter dateConverter) {
+  public TrustedIntermediaryDashboardView(
+      ApplicantLayout layout, DateConverter dateConverter, ProgramRepository programRepository) {
     this.layout = checkNotNull(layout);
     this.dateConverter = checkNotNull(dateConverter);
+    this.programRepository = checkNotNull(programRepository);
```

Comment:
Note:  The reason I'm injecting the ProgramRepository is so that I can get the program names for the cards.  This is from Danielle's refactor last week where we go through the repo rather than getting the name directly from the ApplicationModel.